loopchain/blockchain/invoke_result.py

```diff
@@ -260,10 +266,10 @@ def prepare_invoke(self, block_height: int, block_hash: Hash32) -> PreInvokeResp
 
         return PreInvokeResponse.new(pre_invoke_result)
 
-    def invoke(self, block: 'Block') -> InvokeData:
+    def invoke(self, block: 'Block', tx_versioner: 'TransactionVersioner') -> InvokeData:
```

Comment:
> This is not a review.

I sense that "invoke needs tx_versioner" is a kind of.. a sign for improvement.

But I understand what you want, and it is the best way to supply tx_versioner here I think.
I have no idea to invoke without the versioner. 😢